charter.md

```diff
@@ -85,15 +93,121 @@ are replacing, regardless of the length of that remainder.
 
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 4 = 1.75
+* 1.75 * 3 = 5.25
+* Round up to the nearest whole number (6)
+* Six (6) affirmative votes would be required to remove a member through a vote
+  of no confidence
+
+The call for a vote of no confidence will happen in a public Steering Committee meeting and a
+GitHub issue in kubernetes/steering by a current member and must be seconded by
+another current member.
+
+Once a vote of no confidence has been called, the committee will notify the
+community through the following channels:
+
+* the [community mailing list][dev-list]
+* the [Steering Committee public mailing list][steering-public-list]
+
+This notification will include:
+
+* a link to the aforementioned GitHub issue
+* a short statement from non-affected, seated committee members providing
```

Comment:
I would suggest this statement be agreed to by the member making the motion, and the seconder. This way it represents more than one voice, but at the same time, doesn't require all six folks to bikeshed on the statement prior to a vote being called. The remaining members would agree to this statement when they vote by voting in favour of it.

Reply:
sounds good

Reply:
Fixed in https://github.com/kubernetes/steering/pull/248/commits/4e46d388f05fd34c6c2527fa402666cb55913dc7.